### PR TITLE
Feature/component logo

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Please review the logo component changes and let me know your comments

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/content/hclecomm/us/en/HomePage/.content.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/content/hclecomm/us/en/HomePage/.content.xml]: @DeepaliAgrwl  Why home page checkedin as  a part of logo component?

Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/content/hclecomm/us/en/HomePage/.content.xml]: I created it for testing. But yes this should not be a part of testing. I have removed it from this branch and checked in the code as well

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml [hunk 2]
```diff
@@ -7,12 +7,23 @@
         <foundation jcr:primaryType="nt:unstructured">
             <components jcr:primaryType="nt:unstructured">
                 <responsivegrid jcr:primaryType="nt:unstructured">
+                    <policy_97222330617899
+                        jcr:lastModified="{Date}2020-04-21T14:02:13.092+05:30"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml]: @DeepaliAgrwl  Why this pull request have policy and template related files?

Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml]: somehow the changes of template were merged on my local. Reverted back from this branch. Please have a look

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[hclecomm.logo,hclecomm.structure]"/>
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml]: @DeepaliAgrwl  Keep only one category -hclecomm.logo and update other files accordingly

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/.content.xml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/.content.xml]: @DeepaliAgrwl  Why this file is part of logo comp?

Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/.content.xml]: reverted back this file

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml (hunk 3 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml]: @rakeshkumar-verma removed one category. keeping one category as heclecomm.logo

Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml]: also changed the respective base-clientlib for this

**Pull request closed without merging**